Source/WebKit/WebProcess/WebCoreSupport/WebDragClient.cpp

```diff
@@ -44,6 +44,26 @@ void WebDragClient::willPerformDragDestinationAction(DragDestinationAction actio
         protectedPage()->mayPerformUploadDragDestinationAction(); // Upload can happen from a drop event handler, so we should prepare early.
 }
 
+
+void WebDragClient::willPerformDragDestinationAction(DragDestinationAction action, const DragData& dragData, FrameIdentifier frameID, CompletionHandler<void()>&& completionHandler)
+{
+#if PLATFORM(COCOA)
+    protectedPage()->fetchSandboxExtensionsForDragAction(frameID, [page = RefPtr { m_page.get() }, action, completionHandler = WTFMove(completionHandler)] () mutable {
+        if (!page) {
```

Comment:
This null check can never fail.  It would've already crashed when calling fetchSandboxExtensionsForDragAction on null or it would not be null here.